src/backend/symbols.ts

```diff
@@ -89,7 +89,7 @@ export class SymbolTable {
     }
 
     public getStaticVariables(file: string): SymbolInformation[] {
-        return this.symbols.filter((s) => s.type === SymbolType.Object && s.scope === SymbolScope.Local && s.file === file);
+        return this.symbols.filter((s) => s.type === SymbolType.Object && s.scope === SymbolScope.Local && !s.name.includes(".") && (s.file === file || file.endsWith(`/${s.file}`)));
```

Comment:
So, if I have a duplicate file name like `myfile.c` in two different directories, we could get a match for both. I am okay with that but it could be confusing if/when it really happens. Do we know the root cause of why this is even happening?